shared/chat/conversation/info-panel/bot.tsx

```diff
@@ -65,6 +66,11 @@ export const Bot = (props: BotProps) => {
               {usernameDisplay}
               {lower}
             </Kb.Box2>
+            {showInChannel && (
```

Comment:
I was thinking this would just always be on the screen, and be more in the negative direction. That is, there would be a label indicating a bot is not in the channel as part of the row.

Reply:
gotcha, so more like the add to channel button but without the action associated